src/ConnectedMode.UnitTests/Migration/FileCleanerTestCases/Ruleset_IncludedTopLevel-project-key_Input.ruleset.xml

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This test didnt need to be a ruleset I suppose, it could have been a project file aswell. It is just for testing that we do not remove the root element.

Reply:
The problem won't happen with project files as we never remove nodes that are directly parented under the root node. We only remove:

```
<Project>
    <PropertyGroup>
        <CodeAnalysisRuleSet> <-- could be removed
...
```

or 

```
<Project>
    <ItemGroup>
        <AdditionalFiles Include="...">   <-- could be removed
...
```

```